ci/build_container/update_build_container.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/bash
+read -r -p "Do you have master checked out with most recent changes? [y/N] " response
+if [[ $response =~ ^([yY][eE][sS]|[yY])$ ]]
+then
+  TAG="$(git rev-parse origin/master)"
```

Comment:
Agreed, that way there is no divergence between the tag and what was actually built.